// react-router/dist/development/index-react-server-client.js
"use strict";Object.defineProperty(exports, "__esModule", {value: true});/**
 * react-router v7.18.1
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */
"use client";










var _chunkG5KIBF6Ujs = require('./chunk-G5KIBF6U.js');














var _chunkSA4DP3SFjs = require('./chunk-SA4DP3SF.js');























exports.BrowserRouter = _chunkG5KIBF6Ujs.BrowserRouter; exports.Form = _chunkG5KIBF6Ujs.Form; exports.HashRouter = _chunkG5KIBF6Ujs.HashRouter; exports.Link = _chunkG5KIBF6Ujs.Link; exports.Links = _chunkSA4DP3SFjs.Links; exports.MemoryRouter = _chunkSA4DP3SFjs.MemoryRouter; exports.Meta = _chunkSA4DP3SFjs.Meta; exports.NavLink = _chunkG5KIBF6Ujs.NavLink; exports.Navigate = _chunkSA4DP3SFjs.Navigate; exports.Outlet = _chunkSA4DP3SFjs.Outlet; exports.Route = _chunkSA4DP3SFjs.Route; exports.Router = _chunkSA4DP3SFjs.Router; exports.RouterProvider = _chunkSA4DP3SFjs.RouterProvider; exports.Routes = _chunkSA4DP3SFjs.Routes; exports.ScrollRestoration = _chunkG5KIBF6Ujs.ScrollRestoration; exports.StaticRouter = _chunkG5KIBF6Ujs.StaticRouter; exports.StaticRouterProvider = _chunkG5KIBF6Ujs.StaticRouterProvider; exports.UNSAFE_AwaitContextProvider = _chunkSA4DP3SFjs.AwaitContextProvider; exports.UNSAFE_WithComponentProps = _chunkSA4DP3SFjs.WithComponentProps; exports.UNSAFE_WithErrorBoundaryProps = _chunkSA4DP3SFjs.WithErrorBoundaryProps; exports.UNSAFE_WithHydrateFallbackProps = _chunkSA4DP3SFjs.WithHydrateFallbackProps; exports.unstable_HistoryRouter = _chunkG5KIBF6Ujs.HistoryRouter;
